src/models/ComponentAttribute.js

```diff
@@ -84,6 +84,37 @@ class ComponentAttribute {
       || this.__ruleRegexValidation();
   }
 
+  /**
+   * Get the reference value of the attribute.
+   * @returns {(string | boolean | number | Array | ComponentAttribute[])}
+   * The value of the attribute.
+   */
+  getReferenceValue() {
```

Comment:
Is it not already  possible to access `this.value` without a function ? If not, is it not a better solution to use getter & setter instead, like above `isVariable?`

Reply:
We can access, but the `Reference` value need to be overriden by other plugin like Terraform